fuzzer.js

```diff
@@ -0,0 +1,52 @@
+const micromark = require('./index')
+const frontmatter = require('micromark-extension-frontmatter')
+const gfmSyntax = require('micromark-extension-gfm')
+const gfmHtml = require('micromark-extension-gfm/html')
+
+function fuzz(buf) {
+  try {
+    // // commonmark buffer without html
+    micromark(buf)
+
+    // // commonmark buffer with html
+    micromark(buf, 'utf-8', {
+      allowDangerousHtml: true,
+      allowDangerousProtocol: true
+    })
+
+
+    micromark(buf, 'utf-8', {
+      extensions: [frontmatter()]
+    })
+
+    micromark(buf, 'utf-8', {
+      extensions: [frontmatter()],
+      allowDangerousHtml: true,
+      allowDangerousProtocol: true
+    })
+
+    // gfm buffer without html
+    micromark(buf, 'utf-8', {
+      extensions: [gfmSyntax()],
+      htmlExtensions: [gfmHtml]
+    })
+
+    // gfm buffer with html
+    micromark(buf, 'utf-8', {
+      allowDangerousHtml: true,
+      allowDangerousProtocol: true,
+      extensions: [gfmSyntax()],
+      htmlExtensions: [gfmHtml]
+    })
```

Comment:
these could potentially be moved the extension repo rather than on micromark itself. :man_shrugging: